### ISSUE TITLE
Race Audit

### ISSUE THREAD

**Issue by queryluke** [queryluke/masseffect-5e#236]
For discussion of #235, I decided on doing a quick race audit.

Feats: 6 points
SR Trait: 4 points if has short rest regen
LR Trait: 3 points if it has long rest regen
Minor: 2 points. Additional proficiency, ac bonus, speed bonus, vision type, resistances or immunity*
Flavor: 1 point. Unique to race with limited applications

* Depending on the resistance/immunity this could either be a major or considered a Feat

Quickly auditing the classes:
* Asari: 9 (counting cantrip as a 4, but could be argued otherwise, 4 points for skill versatility)
* Batarian: 10
* Drell: 11 (1 point for 2m speed boost)
* Humans: 12
* Geth: 15
* Krogan: 11
* Quarian: 12 (only 1 for hermetic suit even though there are 2 resistances, but risk involved. Also 3 points for the additional ability score)
* Salarian: 12 (4 points for photographic memory, as it doesn't require the skill to already be proficient)
* Turian: 12 (counting Turian toughness as a feat)
* Volus: 12 (savvy dealings could be 1 or 2, I counted as 2)
* Vorcha: ~9 (speed boost, considering Hungry Jaws and Bloodlust and 1 feat for 5,  2 for bite)
* Unshackled AI: 17 (definitely needs a nerf)

Overall, it's a nice balance. But Asari and Vorcha could justifiably use some additional traits. Drell, Krogan, and Batarians could use a minor trait or a bit of retooling. We've discussed a possible Drell improvement in #185

**Comment by bpmcpherson** [comment]
#235 Discusses some additions to the Vorcha.

**Comment by TheMarmotKing** [comment]
Getting seem to be a little on the opposite side.  Maybe impose disadvantage on certain kinds of charisma rolls?

**Comment by queryluke** [comment]
> Geth seem to be a little on the opposite side. Maybe impose disadvantage on certain kinds of charisma rolls?

I'm fine with the high score of Geth because they can't ever take feats.

**Comment by Nmmoriarty** [comment]
The 1d6 per 10 minutes Vorcha's healing element is very strong.  Effectively the Vorcha gets an average 21 hit points every hour, which is a lot of healing.  Effectively, until level 11+ play, the Vorcha will never need to spend hit dice -- but the rest of the party will.  The party will have to stop despite the Vorcha being able to carry on, unless the Vorcha is face taking all the damage somehow.  It also is a bit out of line with the way that that 5th ed does healing 

While I do think the Vorcha should get a healing element, I think it should come from a "free" number of hit dice per short rest (1 hit dice to start, increasing to 2d6 at later levels) or a flat bonus to hit dice equal to their CON modifier that gets better as they level up.  This way, instead of just making impossible to seriously damage a Vorcha unless they go from full to dead in one encounter, the Vorcha gets a good healing element that allows the party to save resources -- they spend less medi-gel and other resources to get the party back in fighting condition -- but the Vorcha still take substantive damage and aren't effectively unkillable.

Speaking of healing elements, we could add an ability to the Krogan that gives them the ability to negate a critical hit once per long rest due to their redundant systems.  This helps bolster a Krogan's ability to tank hits by reducing the damage taken, rather than the Vorcha's ability to recover damage, making their flavor different.

On the subject of subraces, I think it is an idea worth exploring, but not overdone.  The Vorcha could easily get subraces since they adapt to their environment, natural born Krogan and tankbred makes sense, and maybe the Asari, but most other species don't really have this kind of concept.  However, I don't think the Geth should get a subrace -- Heretical Geth are, effectively, thralls of the Reapers, so it makes little sense to a playable race (unless you're playing an indoctrinated party, and even then the only difference between Consensus and Heretical Geth is a simple math error, I doubt that warrants a separate subrace).

**Comment by bpmcpherson** [comment]
I took inspiration for the regeneration from the [Ring of Regeneration](https://open5e.com/equipment/magic-items/ring-of-regeneration.html) which I found as part of 5e materials, although I do see the point of it being too much. 

Regarding Krogan, maybe their melee attack can be like their charge from the multiplayer, where it knocks enemies around easily?

**Comment by Nmmoriarty** [comment]
Yeah, the Ring of Regeneration is a _Very Rare_ quality magic item, available only to the luckiest of adventurers.  Giving every Vorcha that power is just way too much.

As far as the knockdown on Krogans, it is a good idea, but I'm not sure I am sold on it.  It could get complicated, and knocking targets prone isn't necessarily the best tactical option.  I still think the ability to negate a critical hit fits more in line with the Krogan's ability to soak a ton of punishment (on top of the AC bonus and Deathless ability).

**Comment by bpmcpherson** [comment]
Yeah, I'm aware of that. It was more along it being the only source I could find that had a regeneration effect that wasn't a spell. 

But I was thinking having that along with the critical hit negate, since both are Krogan-like effects. Or at the very least, a bonus to the Shove Action

> Shoving a Creature
Using the Attack action, you can make a special melee attack to shove a creature, either to knock it Prone or push it away from you. If you’re able to make multiple attacks with the Attack action, this attack replaces one of them.

The target must be no more than one size larger than you and must be within your reach. Instead of making an attack roll, you make a Strength (Athletics) check contested by the target’s Strength (Athletics) or Dexterity (Acrobatics) check (the target chooses the ability to use). If you win the contest, you either knock the target Prone or push it 5 feet away from you.

**Comment by Nmmoriarty** [comment]
Volus' Savvy Dealings somewhat locks anyone playing them into being the team's designated shopper.  It is a tricky game balance thing, especially since this system differs from DnD 5th substantially in this arena: there are actually things to spend your hard earned gol-- I mean, credits on.  On top of that, given the games infrastructure, refilling your consumables is more an issue of access than an issue of cost.  My original thought was their Savvy Dealings ability would drop the difficulty of acquiring rarer items, but this can have some unintended consequences.

- Scenario 1 is that the campaign is a "scum and villainy" style where the players are not members of a military outfit, so it becomes a requirement to have a Volus in the party if you want to reliably have access to military grade gear (though grenades, in their current state, start to lose their potency after level 6, but that's another issue).

- Scenario 2 is you are a military outfit (as is traditional for ME games), and access to military class gear is paltry.  Volus characters effectively won't use this trait.

I'm not sure I like how either shakes out.  The other alternatives is that Volus start with more credits (but to balance that we'd have to look at what the expected wealth curve per level is), or a solid discount on buying things (which makes them designated shopper).  If we go discount, I think we need to add a limiter that they can get an item 5% to 15% (again, not sure on the wealth curve) cheaper, but only once per week/month so players can't abuse it.

**Comment by Duneday** [comment]
Asari could get a Racial to make their Biotic spell modifier Cha instead of Wis. This would make sense because the power to use biotics is an innate ability for Asari while other races tend to need to develop the ability. This could also be a step towards sealing the Cha void in the game, although ultimately it would pigeonhole anyone wanting to be Cha caster into being an Asari, but its a step. 

While given the chance the Vorcha healing could be quite useful, in the midst of combat its likely to be of no real use. In fact, Unshackled AI's Repair Matrix is far more powerful and, in my opinion, useful. I would feel safe removing the healing factor from a mechanics standpoint as it takes a longer period of time for it to be useful usually taking a week to heal major wounds. Instead we could focus on the other adaptive elements of their biology maybe keeping a limb regeneration in order to maintain some semblance of the healing.  The Vorcha are rapidly adaptive and with their cellular regeneration they also have an immunity to disease which is noticeably absent. Their adaptations to environment and damage could be a substantial replacement to the healing, maybe either giving an immunity or resistance to environment based hazards, or a resistance to the first type of damage they take in combat.

Batarians, I agree, could use a proficiency in Insight, but I believe intimidation could also be a viable option because, and Im quoting here, "they often gain the upper hand in interspecies arguments because other races find it difficult to know which eyes to focus on when speaking to them." They may also get an option to get starting credits equivalent to Volus as money is a major factor in their culture. 

The drell are pretty good, although as a species they have an eidetic memory which is missing. Also despite demonstrably being able to use both biotics and tech they are not able to be sentinals, which is weird and I believe an oversight. 

Humans are typical a little boring, but balanced. 

Geth and Unshackled AI, for the time being, I will skip.

Krogan are good, and I agree with Nmmoriarty on the ignore a single critical, probably once per long rest, to mirror their redundant physiology and give them that extra little bit of tankiness.  To better mirror their biology I would argue dropping their proficiency in intimidation in favor of a proficiency in perception due to their 240 degree vision. Also even though it is not likely I believe that Krogan do possess the ability to become infiltrators it is simply not something their culture would value highly as a skillset favoring more direct confrontation. 

Quarians are pretty solid, I would argue that maybe their Attribute bonuses could be tweaked. Charisma and Wisdom don't seem particularly "Quarian." In fact their major enemy (Geth) were born from a lack of both. That said Intelligence and dexterity are both good options as they are obviously intelligent and dexterity is a necessity to practiced engineers.

For Salarians I already see the impact my coplayers have had with the inclusion of the Metabolism, so good job there ^_^. Otherwise they seem fine for the time being.

Turians are pretty much fine. Maybe add a perception proficiency for their eyesight to offset the lack of choice that human provides.

Volus are an interesting point, but ill start with the concrete facts. While the gravity from their planet is roughly 1.5x earth it is explicitly mentioned that "they are not physically adept compared to most species, volus mostly make their influence felt through trade and commerce." Making the justification for a +1 str a little flimsy. A Cha bonus would be more lore friendly if not mechanically stunted (Cha Void) or maybe even an Int bonus would be more suiting. Savvy dealings could be great it could be awful, either way it does need a solid mechanic. I have to disagree with Nmmoriarty here though, in purely meta sense he would be correct and Volus could become a designated shopper, but that completely ignores character which should always be a driving force in games. I believe a Volus would be a little resentful most of the times and probably derive no small amount of enjoyment in getting better deals than those around him (a bit of a napoleon complex). Ultimately I'm of the belief that the solid discount is the best option even in meta situations of "designated shopper" as a racial it may accelerate party growth slightly, but it is still at the behest of a GM as to what becomes available when and how much money a party has access to in addition to what equipment they would have access to. That said the point of the suit drawback is a valid one, it could function effectively as the Quarian one requiring an action to seal after an unprotected crit.

I do believe subraces would be a nice addition, although I believe getting the base races down would be best and then if one seems too powerful or the lore would demand it maybe break up some binuses into subraces. Additionally, there is precedent for races to have choices of secondary attribute bonuses so something like Volus, for example, could gain +2 Wis and then have a choice between +1 Cha or +1 Int. This could make numerous races more desirable and custom to the player. 

Lastly there are a few races (Salarians, Asari, and Drell) that have the ability to see Ultraviolet, while maybe not a mechanical advantage it is worth mentioning. Potentially this could mean seeing through invisibility, although that could be quite powerful. Maybe putting a range on seeing the invisibility could be a suitable balance with Asari seeing it form a greater distance and Drell being closer. Just a thought.

**Comment by Duneday** [comment]
After posting a friend brought up a good point about Quarians and im going to quote him here, "they are highly isolationist when not on their pilgrimage Very few outsiders are allowed access into the fleet ships
you only see the youngs ones out to prove themselves and bring something back to benefit their ppl
so I think a similar stat curve to salarians fits them better +2 int, +1 dex or flip those"

**Comment by damuttley** [comment]
Ok, so I was discussing with Dune and here's what I came up with for race tweaks/changes:

1.  The above post on Quarians was me, so I won't restate that.

2.  Drell do have the eidetic memory, though they would be forced to undergo will saves for reliving longer memories, increasing in DC as time progresses.  They can become lost in reliving a moment, with longer memories being more likely for this to happen.

3.  Asari could become the sorcerors of the ME5e and use CHA, since there is a gap in classes/abilities that utilize that stat.  Might be worthwhile to examining giving them additional potency in biotics with either increased save DCs for them or giving them bonus biotic points.  

4.  Vorscha regeneration takes time, and as such wouldn't be hugely viable for a campaign.  Its not Piccolo regenerating his arm, but more like a starfish.  Environmental resistance bonuses seems the most likely area to improve them.  They are practically vermin that live in squalor where no other sentient being would live.  Depending on their home environment, you could generate a table or talk with the DM and create bonuses for them.  Like Vorscha living in a toxic area becoming immune to poison fumes or Vorscha living in other types or extremes gaining bonuses to reflect their adaptability.  You can even change these, IF you spend enough time in an area (have to be fairly significant to override what they already have).

5.  I dislike Volus as a playable race.  They are going to be pigeonholed due to fluff and applicable bonuses as stated in the previous posts.  Volus are not great fighters.  You rarely see them as anything other than merchants and traders. They would have reduced speed due to size, their suits are required for them to be able to breathe, they are not great at melee combat.  Shooting would work for them, but why play a Volus to shoot when you have so many more races?  I am of the opinion that Volus should be removed as a playable race and kept as NPCs.  

6.  Krogans ignoring a crit per long rest sounds good.  As does the proficiency in perception.  Hard to sneak up on those jumpy toads and their huge field of view.  Though they should lose this bonus when battleraging (too much focus on kersplatting to notice things).

**Comment by Nmmoriarty** [comment]
Holy wow I totally forgot about AI.  Yeah, I am not terribly fond of AI (or, really any "snowflake" races, honestly), that healing factor they have is really good, giving you an average 35 hit point recovery in a minute.  That's really strong, even into higher levels.  If we want an emergency repair, I'd say give it a 2d6 once per day reaction, increasing it as you go up in levels.  Good in a pinch, but won't bring you back up to full.

While I do think Vocha should get a regenerate lost limbs mechanic, some mechanic benefit of healing (and don't forget that a long rest restores all your HP and half your HD).  Hell, maybe even they restore all of their HD on a long rest so they are more inclined to take damage, since they've got more resources to restore it.  All options worth considering.

For the Volus, a solid discount is fine, but I still feel like it breaks game balance too much, especially if the discount is high.  It needs a limiter of some variety to keep players from being 5% to 20% richer simply because there is a Volus around -- and without a limiter, every party will want a Volus.  Volus will be essentially left in support roles, but that's not super bad, but a little niche from a design perspective.

On the concept of Asari, making them switch their biotic casting stat to CHA regardless is less... useful, in my opinion that just giving them a +2 CHA and +2 WIS or something else.  The only class this has any effect on is Adept, so it really doesn't help that much, since if you're going to max your casting stat anyway, so why bother?

Alternatively, (this more belongs in #203 but it is relevant here) you could make a species' biotic power run of a certain stat rather than it being based on classes (Humans use WIS on biotics, Asari use CHA, etc.), but this seems a pretty inelegant and obtuse solution to the CHA void problem.

**Comment by Duneday** [comment]
I dont think that the Healing on AI is super overtuned. It could be brought down a little, but its not too outrageous in my opinion. 

Vorcha Healing is a longer term thing taking like a week which is why id be fine seeing it reduced to a limb regeneration although I do like your idea of the full HD refund during a long rest, maybe even losing all exhaustion levels after a long rest, it gets a bit unnecessary at that point though.

In terms of Volus, its still a GM issue at that point Stores have stock and its not likely theyll keep super expensive things on hand, such things usually need to be ordered and pass through red tape, there are sooo many options for a GM to reign in spending its kind of a nonissue. As characters they seem like theyd have issues fitting into a combat squad. From my perspective being a short fat money grubber doesn't appeal as a player.

Having a Cha based caster on a race with a natural cha would be invaluable. The synergy, lore and mechanics, makes sense. Also, having even a single race/class combo that uses Cha is almost a must otherwise Cha may as well not be a thing. Theres more than combat in a game and skills should always be something you can utilize in addition to combat prowess, without a Cha caster those skills would generate a onesided character that would literally be about useless outside social situations.

**Comment by Duneday** [comment]
I also agree the solution is a little inelegant, but I believe the root of the problem is the spellcasting system itself. While it makes perfect sense in the lore it doesn't lend itself to the DnD stat archetype. Breaking down all casting into the two types of Tech/Biotic and blanketing them into one stat just doesnt meld with the tabletop system well. Making the spell modifiers based on Race isnt ideal in my opinion, but due to Mass effects limited Class system its difficult to justify making it class based as well, especially when class choice is limited based on Race anyways.

**Comment by Nmmoriarty** [comment]
Regrettably, I have to disagree on the viability of Volus being combatants -- ME3 Multiplayer has four Volus classes you could be, and they are pretty absurdly good support characters.  So while I admit it won't be a popular option, it being an option isn't canonically off the table.  It should be a little off the wall, but it isn't outside the realm of possibility.

Also, you are aware that currently is it proposed to change Vanguard and Sentinel to CHA, so giving the Asari always cast on CHA only affects Adepts?  My problem isn't using CHA to cast, it is giving Asari an ability that really only matters in one instance.

**Comment by Duneday** [comment]
Im aware of the suggested VAN and SEN, but it has issues as well and Im operating these based on the state of the book, which has no Cha casters.
I could see Volus in combat and it is in the mutiplayer so the precedence is there. My argument is more one of desirability not viability.

**Comment by queryluke** [comment]
Didn't realize this was going to spark such a huge discussion! There are a lot of great ideas hear and I'll break this out into multiple issues so we can drill down into each race specifically.

Of note to the people who've mentioned Drell's eidetic memory, we started an issue a while back about that #185, so feel free to take ideas there.

On the issue of removing races (AI and Volus have been tossed around), I'll say that's not something I plan on doing. As long as a race is balanced and doesn't outshine any other race, it's up to the GM and their players to decide if they want to use it. The same sentiment applies to adding races. If someone comes up with a balanced race concept, I'll more than likely include it. The caveat would be a race that either 1) doesn't exist or 2) is demonstrably unfit to be a player character (yes, I'm leaving a bit of wiggle room here). 

However,  since the system is going so rapidly (already 3 proposed races and one subrace), I'm more than happy to break the races out into categories. So we can certainly have "Standard" and "Expanded" races. I'll make a new issue to "straw poll" which races belong where.

**Comment by queryluke** [comment]
Thinktanks for race changes:

* [Asari](https://github.com/queryluke/masseffect-5e/issues/241)
* [Batarians](https://github.com/queryluke/masseffect-5e/issues/243)
* [Drell](https://github.com/queryluke/masseffect-5e/issues/185)
* [Krogan](https://github.com/queryluke/masseffect-5e/issues/242)
* [Quarians](https://github.com/queryluke/masseffect-5e/issues/245)
* [Turians](https://github.com/queryluke/masseffect-5e/issues/246)
* [Volus](https://github.com/queryluke/masseffect-5e/issues/244)
* [Vorcha](https://github.com/queryluke/masseffect-5e/issues/235)

There was additionally a lot of discussion on subraces. Please feel free to on a new issue of a subrace idea.
